sbncode/EventGenerator/MeVPrtl/Tools/HNL/HNLMakeDecay_tool.cc

```diff
@@ -181,7 +250,7 @@ HNLMakeDecay::ThreebodyMomentum HNLMakeDecay::isotropic_threebody_momentum(doubl
 
   // The azimuthal angle of B and C about A is distributed uniformly
   double gammaB = (2*GetRandom() - 1.) * M_PI;
-  double gammaC = -gammaB;
+  double gammaC = fmod(gammaB + 2*M_PI, 2*M_PI) - M_PI;
```

Comment:
Is this a physic change? Would it destroy backward compatibility? Do we care if it does?

Reply:
It is a physics change, it's fixing a bug. 

As far as backwards compatibility, I don't think I've shown results before with 3-body decays. When I tried to for the results in this PR, I found the bug. The simulation makes unphysical results otherwise.